packages/Tabs/src/components/Tab/Tab.js

```diff
@@ -42,6 +42,7 @@ export default function Tab(props) {
       <a
         css={tabStyles}
         className="tab tab-link"
+        data-pka-anchor="tab"
```

Comment:
lets differentiate this by saying tab-link

Reply:
done